apps/styleguide/src/forms/InputPassword.vue

```diff
@@ -13,6 +13,7 @@
       :aria-describedby="id + 'Help'"
       :placeholder="placeholder"
       @input="emitValue"
+      @keydown.enter="$emit('enterPressed')"
```

Comment:
For it to reliably to work, it's best on keyup

Reply:
Nu uses keyup